README.md

```diff
@@ -106,6 +107,17 @@ For a comprehensive list with many more clients along with screenshots, the
 - [Matrix CRDT](https://github.com/yousefED/matrix-crdt) - Integrates CRDTs like
   [Yjs](https://github.com/yjs/yjs) with Matrix for storage and sync. `MPL-2.0`
   `TypeScript`
+  
+## Collaborative Maps
+
+- [ODIN](https://odin.syncpoint.io) - [Open Source](https://github.com/syncpoint/ODINv2)
```

Comment:
To match the format of the other entries, let's move this repo link to end styled as `([Repo](https://github.com/syncpoint/ODINv2))`.